controllers/preview-pane.js

```diff
@@ -0,0 +1,56 @@
+const dom = require('@nymag/dom'),
+  _ = require('lodash'),
+  previewSizes = {
+    small: { w: 375, h: 660 },
+    medium: { w: 768, h: 1024 },
```

Comment:
our breakpoint is 600 (for the most part), but I think the most common "tablet-ish" screen width is 768